static/js/murr_list.js

```diff
@@ -42,8 +42,11 @@
                     $slideIn = $('.js-murr-detail'),
                     $slideInOverlay = $('.js-murr-detail-overlay');
 
+                $('.js-murr-detail-body', $slideIn).html("");
                 $slideIn.addClass('modal-box_closed');
                 $slideInOverlay.addClass('modal-overlay_closed');
+                window.location.reload(true);
```

Comment:
а зачем нам здесь перезагружать текущую страницу?

Reply:
Сделал для обновления MurrList

Reply:
```suggestion
                window.location.reload(true);
```
Я бы заменил на $('.js-murr-detail-body').html('');

Reply:
Изначально планировалось все-таки ходить аяксом за новым мурр листом. Но если перезагрузка сейчас всех устроит, то можно оставить и так. Запустил обсуждение в тг